PyNite/Member3D.py

```diff
@@ -1186,6 +1186,92 @@ def PlotDeflection(self, Direction):
         Member3D.__plt.xlabel('Location')
         Member3D.__plt.title('Member ' + self.Name)
         Member3D.__plt.show()
+    
+#%%   
+    def RelativeDeflection(self, Direction, x):
+        
+        d = self.d()
+        d1y = d[1,0]
+        d2y = d[7,0]
+        d1z = d[2,0]
+        d2z = d[8,0]
```

Comment:
Use `dyi`, `dyj`, `dzi`, and `djz` for member beginning and ending points. This will keep it consistent with the rest of the code. I've been using `i` and `j` for member ends and `1` and `2` for segments ends.